### PR TITLE
✨Source Stripe: New Stream - ShippingRates

### LINKED ISSUE #26481: Source Stripe: new stream Shipping Rates
### What area the feature impact?

Connectors

## Tell us about the problem you're trying to solve
[Shipping Rates](https://stripe.com/docs/api/shipping_rates)
Shipping rates describe the price of shipping presented to your customers and can be applied to [Checkout Sessions](https://stripe.com/docs/payments/checkout/shipping) and [Orders](https://stripe.com/docs/orders/shipping) to collect shipping costs.
[The shipping rate object](https://stripe.com/docs/api/shipping_rates/object)

## Describe the solution you’d like
Research pros and cons for implementing this stream

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #26481

## Enhancement
New streams: 

- Shipping Rates

## Recommended reading order
1. `source_stripe/source.py`
2. `source_stripe/streams.py`
3.  `source_stripe/schemas/*`

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed
+

### CODE REVIEW

**Comment by sajarin** [comment]
Can you resolve the merge conflicts here?

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/shipping_rates.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "object": {
+      "type": ["null", "string"]
+    },
+    "active": {
+      "type": ["null", "boolean"]
+    },
+    "created": {
+      "type": ["null", "integer"]
+    },
+    "delivery_estimate": {},
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/shipping_rates.json:19]: I don't think this is a valid typing, please update it

File: docs/integrations/sources/stripe.md [hunk 2]
```diff
@@ -93,6 +94,7 @@ The Stripe connector should not run into Stripe API limitations under normal usa
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                                              |
 |:--------|:-----------|:---------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| 3.7.0   | 2023-06-09 | [27217](https://github.com/airbytehq/airbyte/pull/27217) | Add `ShippingRates` stream                                                                                                                           |
```
Comment (davydov-d) [inline docs/integrations/sources/stripe.md:97]: please bump version here

**Review by davydov-d — CHANGES_REQUESTED** [review]
Left some minor comments, but please also update the connector version in the Dockerfile and metadata.yaml and resolve the conflicts, thanks!

**Review by davydov-d — APPROVED** [review]

**Comment by davydov-d** [comment]
@btkcodedev you'll have to also run code formatting and fix unit tests please
`python -m black --config=/pyproject.toml --check .`
`python -m pytest unit_tests`

**Comment by btkcodedev** [comment]
@davydov-d I've fixed unit tests, but can't help with code formatting as I am facing some issues with the usage of the command.
Could you please check

**Comment by davydov-d** [comment]
@btkcodedev please run `./gradlew airbyte-integrations:connectors:source-stripe:airbytePythonFormat` from the project root instead

**Comment by btkcodedev** [comment]
@davydov-d Ran code formatting, Fixed Unit tests, Cleared conflicts

**Comment by btkcodedev** [comment]
@davydov-d I am using one of the forks of master for the PRs, I could not change the remote inorder to rebase from your branch, Could you cherrypick changes and push to this PR?
Sorry for that :)

**Comment by davydov-d** [comment]
> @davydov-d I am using one of the forks of master for the PRs, I could not change the remote inorder to rebase from your branch, Could you cherrypick changes and push to this PR? Sorry for that :)

@btkcodedev yep, working on that. A bit messed up, but will figure it out soon

**Pull request merged**